system test are failing with latest python spanner release

System test is failing with `nox -s system` command.

```
>           raise ValueError(NUMERIC_MAX_SCALE_ERR_MSG.format(abs(scale)))
E           ValueError: Max scale for a numeric is 9. The requested numeric has scale 28
```

#### Environment details

  - Programming language: python
  - OS: Mac
  - Language runtime version: python 3.8
  - Package version:
django-google-spanner                2.2.1b1
google-cloud-spanner                 3.4.0

#### Steps to reproduce

  1. Run nox -s system test

The test raises Value error 
```
FAILED tests/system/django_spanner/test_decimal.py::TestDecimal::test_decimal_precision_limit - ValueError: Max scale for a numeric is 9. The requested numeric has scale 28
```

--- a/tests/system/django_spanner/test_decimal.py
+++ b/tests/system/django_spanner/test_decimal.py
@@ -89,12 +89,10 @@ class TestDecimal(TransactionTestCase):
         """
         num_val = Number(num=Decimal(1) / Decimal(3))
         if USE_EMULATOR:
-            msg = "The NUMERIC type supports 38 digits of precision and 9 digits of scale."
-            with self.assertRaisesRegex(IntegrityError, msg):
+            with self.assertRaises(ValueError):
                 num_val.save()
         else:
-            msg = "400 Invalid value for bind parameter a0: Expected NUMERIC."
-            with self.assertRaisesRegex(ProgrammingError, msg):
+            with self.assertRaisesRegex(ProgrammingError):
                 num_val.save()
 
     def test_decimal_update(self):

--- a/tests/system/django_spanner/test_decimal.py
+++ b/tests/system/django_spanner/test_decimal.py
@@ -7,7 +7,6 @@
 from .models import Author, Number
 from django.test import TransactionTestCase
 from django.db import connection, ProgrammingError
-from django.db.utils import IntegrityError
 from decimal import Decimal
 from tests.system.django_spanner.utils import (
     setup_instance,

